databricks/koalas/frame.py

```diff
@@ -1346,7 +1346,7 @@ def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
               for colname, asc in zip(by, ascending)]
         kdf = DataFrame(self._sdf.sort(*by), self._metadata.copy())
         if inplace:
-            self._sdf: spark.DataFrame = kdf._sdf
+            self._sdf = kdf._sdf
```

Comment:
Variable Annotations is supported from Python 3.6. https://www.python.org/dev/peps/pep-0526/